### PR TITLE
Initial styling pass

### PR DESCRIPTION
**Pull request by lederer**
Comprehensive styling pass on all known screens and states. That said, I may have missed a few things here or there, and will need to do another, smaller pass to tighten things up regardless.

I made virtually no use of centralized styling (`styles.xml`), preferring to configure each component individually. Now that things look approximately how they should, it might make sense DRY this up in a follow-up pass as well.

screenshot_1528748177

screenshot_1528748189

screenshot_1528748194

screenshot_1528748184

screenshot_1528748200

screenshot_1528748208

screenshot_1528748229

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
Event details for events without a destination associated could use a little rearranging:

screenshot_1528816365

**Comment by flibbertigibbet** [comment]
A couple of notes on the templates:
 - Replacing references to `?attr/colorPrimary` with `@color/color_primary` will fix the Android Studio warnings about not being able to find the reference.

 - `android:textAlignment="textStart"` is also making Android Studio grumpy; it says it's expecting `start` instead due to the gravity.

**Comment by flibbertigibbet** [comment]
Testing on a Galaxy Nexus emulator on API 19 (the lowest we support), the details box display displays with a border; it looks like the expand/collapse button may have gotten cut off a bit.

image

File: app/src/main/java/com/gophillygo/app/data/models/Attraction.java [hunk 1]
```diff
@@ -207,7 +207,8 @@ public String getActivitiesString() {
             if (stringBuilder.length() > 0) {
                 stringBuilder.append(dot);
             }
-            stringBuilder.append(activity);
+            String capitalizedActivity = activity.substring(0, 1).toUpperCase() + activity.substring(1);
```
Comment (flibbertigibbet) [inline app/src/main/java/com/gophillygo/app/data/models/Attraction.java]: (Putting a note here to remind myself to revisit this later)

**Comment by lederer** [comment]
Awesome! Will do.

**Comment by flibbertigibbet** [comment]
Opened up #86 to take a swing at fixing up the merge conflicts.

**Comment by lederer** [comment]
Closing in favor of #86 and #95.

**Pull request closed without merging**